### PR TITLE
BaSP-MR-22: Subscription Schema & Populate

### PR DESCRIPTION
**Pull request by keiler-art**

### CODE REVIEW

File: src/models/Subscription.js [hunk 1]
```diff
@@ -5,15 +5,16 @@ const { Schema } = mongoose;
 const subscriptionSchema = new Schema(
   {
     classId: {
-      type: String,
+      type: mongoose.Types.ObjectId,
       required: true,
-      maxLength: 24,
+      ref: 'Class',
     },
```
Comment (nicolasblasco) [inline src/models/Subscription.js:11]: I think we should add: `_id: false` to the classId property

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by LucianoAlarcon — APPROVED** [review]

File: src/controllers/subscriptions.js [hunk 2]
```diff
@@ -2,7 +2,9 @@ import Subscription from '../models/Subscription';
 
 export const getSubscriptions = async (req, res) => {
   try {
-    const subscriptions = await Subscription.find();
+    const subscriptions = await Subscription.find()
+      .populate('members')
+      .populate('classes');
```
Comment (nicolasblasco) [inline src/controllers/subscriptions.js:7]: I think we have to populate `claseId` instead of classes.

File: src/controllers/subscriptions.js [hunk 3]
```diff
@@ -27,7 +29,9 @@ export const getSubscriptions = async (req, res) => {
 export const getSubscriptionById = async (req, res) => {
   try {
     const { id } = req.params;
-    const subscription = await Subscription.findById(id);
+    const subscription = await Subscription.findById(id)
+      .populate('members')
+      .populate('classes');
```
Comment (nicolasblasco) [inline src/controllers/subscriptions.js:34]: I think we have to populate `classId` instead of classes

**Review by nicolasblasco — CHANGES_REQUESTED** [review]
I think we should also check if this works correctly in postman

File: src/controllers/subscriptions.js (hunk 3 above)
Comment (keiler-art) [reply src/controllers/subscriptions.js]: Done!

File: src/controllers/subscriptions.js (hunk 2 above)
Comment (keiler-art) [reply src/controllers/subscriptions.js]: Done

**Review by JuanManuel-Blois — APPROVED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**